utils.zsh/functions/progress

```diff
@@ -0,0 +1,19 @@
+if (( ! ${+_loglevel} )) _loglevel=1
+
+local -a command=($@)
+
+if (( ${+CI} == 0 && _loglevel == 1 )) {
+  local spin='⠋⠙⠹⠸⠼⠴⠦⠧⠇⠏'
```

Comment:
It truly doesn't matter in the grand scheme of thigns, but sometimes there are three dots per glyph and sometimes there are four. Is that unavoidable?

Reply:
I don't think so, the animation is built on these glyphs as-is and makes it look more fluid that way.

Reply:
Technically, one could always use four dots:
```
⠏⠛⠹⠼⠶⠧
```
Or three dots:
```
⠋⠙⠸⠴⠦⠇
```
But both result in shorter animation loops.
```